pkg/collector/corechecks/servicediscovery/module/impl_linux.go

```diff
@@ -82,8 +82,11 @@ type discovery struct {
 
 // NewDiscoveryModule creates a new discovery system probe module.
 func NewDiscoveryModule(*sysconfigtypes.Config, module.FactoryDependencies) (module.Module, error) {
+	config := newConfig()
+	LoadIgnoredComms(config)
```

Comment:
Does LoadIgnoredComms relevant only for linux? If not, why isn't it in `newConfig`?

Reply:
I've moved config reading to discoveryConfig